src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -213,6 +197,42 @@ private void FifoWatcher_Deleted(object sender, FileSystemEventArgs e)
             }
         }
 
+        private bool GetRequiresRootAttach()
+        {
+            if (_miMode != MIMode.Clrdbg)
```

Comment:
This check shouldn't be necessary - we already check in InitStreams to be sure the target is GDB before checking this.